tornado-drivers/opencl/src/main/java/uk/ac/manchester/tornado/drivers/opencl/graal/phases/TornadoBatchGlobalIndexOffset.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (c) 2024 APT Group, Department of Computer Science,
```

Comment:
```suggestion
 * Copyright (c) 2024, APT Group, Department of Computer Science,
```

Reply:
We forgot the comma after the year. Please append all files that have new headers.